t/chaos/kubernetes/README.md

```diff
@@ -25,16 +25,16 @@ There are some yaml files for deploying apisix in Kubernetes.
 
 ### Prerequisites
 
-- use `etcd`, if there is no `etcd` service, please install and set etcd address in `../conf/config.yaml`
+- use `etcd`, if there is no `etcd` service, please install and set etcd address in `../../../conf/config.yaml`
```

Comment:
Let's remove unused files.

Reply:
Do you mean that `README.md` and `service-monitor-for-prometheus.yaml`  under kubernetes folder should be removed?

Reply:
Yes